styles.css

```diff
@@ -884,6 +921,91 @@ i.fa-instagram {
         transform: scale(0);
     }
 }
+
+@media all and (max-width: 1440px) {
+    .testim-headline{
+        font-size: 2.25rem;
+    }
+    .testim .cont div h3{
+        font-size: 3rem;
+    }
+    .testim .cont div p{
+        font-size: 2rem;
```

Comment:
The font-size looks a bit big for this viewport. And when the width increases, the font-size is 1.15em again. In this case, I'd use the CSS function clamp() that allows you to set a font-size that grows with the size of the viewport. 😄